lib/jxl/enc_aux_out.cc

```diff
@@ -100,6 +102,7 @@ void AuxOut::Assimilate(const AuxOut& victim) {
 }
 
 void AuxOut::Print(size_t num_inputs) const {
+#if JXL_DEBUG_V_LEVEL > 0
```

Comment:
Ideally, let's do the same thing here (`if (JXL_DEBUG_V_LEVEL > 0)`)

Reply:
done